tests/test_dynamodb_discovery.py

```diff
@@ -53,6 +52,3 @@ def name():
 
     def test_run(self):
         self.pre_sync_test()
-
-
-SCENARIOS.add(DynamoDBDiscovery)
```

Comment:
Why do we remove these 2 lines?

Reply:
It was removed as it was throwing cci error after updating as per https://github.com/stitchdata/tap-tester/blob/master/reference/circleci_configs.md